lldb/packages/Python/lldbsuite/test/make/Makefile.rules

```diff
@@ -352,7 +352,14 @@ ifeq "$(MAKE_GMODULES)" "YES"
 endif
 
 CFLAGS += $(CFLAGS_EXTRAS)
-CXXFLAGS += -std=c++11 $(CFLAGS) $(ARCH_CXXFLAGS)
+
+ifeq "$(SWIFT_CXX_INTEROP)" "1"
+CXXFLAGS += -std=c++17 
+else
+CXXFLAGS += -std=c++11 
```

Comment:
@Michael137 Could we generally bump this on llvm.org for all tests by default?

Reply:
Think it could be worthwhile. Easy enough to test whether anything breaks